Components/Card/card.css

```diff
@@ -63,11 +97,22 @@ section .example-div {
     box-shadow: 3px 3px 20px 5px var(--black);
 }
 
-.text-overlay-card .overlay-title {
-    color: var(--white);
+.text-overlay-card {
     position: relative;
-    bottom: 4rem;
-    margin-bottom: -4rem;
+    height: 40rem;
+    box-shadow: 3px 3px 20px 5px var(--black);
+}
+
+.absolute-positioned-text {
+    color: var(--alert-error);
+    position: absolute;
+    z-index: 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 25rem;
+    height: 100%;
+    background-color: rgba(197, 185, 185, 0.657);
```

Comment:
use variables for colors if you are using it more than one time [Reason: you don't have to track and change its value everywhere instead could change at one place and see changes everywhere]

Reply:
Yes, Will use variables 👍🏻